enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md

```diff
@@ -107,51 +103,134 @@ users' workload Pods so that containers in those Pods can install RHEL subscript
 2. As a cluster admin with a set of CAs pertinent to various users's workloads, I want to share those CAs with my 
 users' workload Pods so that containers in those Pods can utilize those CAs when performing TLS based communication.
 
-3. See [this proposal](https://github.com/openshift/enhancements/pull/214) for additional user scenarios that could 
-utilize this proposal.
+3. As a cluster admin whose user workloads include kmod based scenarios, I want to share the necessary information to
+facilitate those scenarios across multiple namespaces.
 
 ### Implementation Details/Notes/Constraints [optional]
 
 #### Install
 
-A comparison of installing the implementation of this proposal via OLM vs. CVO was done in another enhancement proposal.  
-The net result was to go with CVO.
+As per the open questions, whether to go with a CVO or OLM based approach, or perhaps some combination, is a question
+with no current consensus on the answer.
 
 Some specifics:
 - During the review for the ["Installation of CSI drivers that replace in-tree drivers](https://github.com/openshift/enhancements/blob/master/enhancements/storage/csi-driver-install.md)
-a question was posed about whether the exact type of CSI driver this proposal is describing would use the method 
+a question posed about whether the exact type of CSI driver this proposal is describing would use the method 
 described in that document (namely OLM).  The recommendation from the [storage SMEs was to go with CVO](https://github.com/openshift/enhancements/pull/139/files#r391761362)
-- In particular, using CVO and having it be a day 1 install operation meets the ease of use targets we are trying to
+was to not use OLM, and use CVO instead.
+- Using CVO and thus making it a day 1 install operation meets the ease of use targets we are trying to
 reach with this proposal
 - Crafting OLM dependencies between this proposal and [related proposals](https://github.com/openshift/enhancements/pull/214)
-as part of an OLM based install at the time of this writing is immature.
+as part of an OLM based install at the time of this writing is immature, at least for some reviewers.
+- However, other reviewers wonder if we are in fact close from having either OLM or CVO operators being able to require
+an OLM operator to be present.
+- Using OLM avoids further increasing the install payload size.
+- Using OLM avoids unexpected hiccups in the feature's installation impacting initial cluster install or cluster upgrade.
+
+#### End to end flow (from creation, to opt-in, to validation, and then injections)
+
+An admin creates a new cluster level custom resource for encapsulating the sharable `Secret` or `ConfigMap`.
+
+
+```yaml
+type: ProjectedResource
```

Comment:
specify the apiVersion here too.  `storage.openshift.io/v1` and negotiate with @openshift/storage for whether they want a sub-group on it.  I'm ok with a stutter to scope it properly, so `ProjectResource` in `projected-resource.storage.openshift.io` is fine with me.

Reply:
added in next push

Reply:
I assume we mean `kind` here?